man/systemd.network.xml

```diff
@@ -700,6 +700,16 @@
           <literal>[IPv6PrefixDelegation]</literal> and the <literal>[IPv6Prefix]</literal>
           sections for more configuration options.
           </para></listitem>
+	</varlistentry>
+        <varlistentry>
+          <term><varname>IPv6PDSubnetId=</varname></term>
+          <listitem><para>Configure a specific subnet id from a received prefix delegation to
+          the interface. Allowed values are hexadecimal from 0 up to 8000000000000000 (2^63)
```

Comment:
my comments regarding 2^63 vs 2^63-1 are not addressed?

Reply:
still

Reply:
I think this should be properly covered now.

I also decided to give this a bit of a makeover with help from @megfault (Thanks!). It should be more obvious what is allowed to set and less long (german ;)) sentences.